src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -1564,6 +1564,73 @@ public int ReceiveMessageFrom(byte[] buffer, int offset, int size, ref SocketFla
             return bytesTransferred;
         }
 
+        public int ReceiveMessageFrom(Span<byte> buffer, ref SocketFlags socketFlags, ref EndPoint remoteEP, out IPPacketInformation ipPacketInformation)
```

Comment:
The new practice is to add xmldocs right in the dotnet/runtime PR. See 9f731887a78545c6b59401ed2eb549e85670cb6d from #45083

Reply:
Hmm, it seems like non of the other methods in this class has it, should I start the it or am I looking at the wrong place?

Reply:
You can check #47230 and #47229 as examples adding new methods to `Socket.cs` together with xml docs.

Just copy existing docs, and modify them according to the new overload:
https://docs.microsoft.com/en-us/dotnet/api/system.net.sockets.socket.receivemessagefrom?view=net-5.0

Reply:
took a stab at the documentation, copied from the existing, but existing is missing some of the exception actually thrown from the method, so a bit unsure if they should be added.